Samples/GettingStarted/Regression_TaxiFarePrediction/TestTaxiTrips.cs

```diff
@@ -0,0 +1,15 @@
+﻿namespace Regression_TaxiFarePrediction
+{
+    internal class TestTaxiTrips
+    {
+        internal static readonly TaxiTrip Trip1 = new TaxiTrip
+        {
+            VendorId = "VTS",
+            RateCode = "1",
+            PassengerCount = 1,
+            TripDistance = 10.33f,
+            PaymentType = "CSH",
+            FareAmount = 0 // predict it. actual = 29.5
```

Comment:
Do we allow this field to be a null or NaN? It could help the user understand this isn't an input but the output calculated value?

Reply:
I don't think I can make it nullable type, getting Type mismatch exception.